bot/cogs/moderation/silence.py

```diff
@@ -13,26 +11,26 @@
 from bot.constants import (STAFF_ROLES, Channels, Colours, Emojis, Guild,
                            Icons, Roles)
 from bot.converters import SilenceDurationConverter
-from bot.utils import time
 from bot.utils.checks import with_role_check
 from bot.utils.scheduling import Scheduler
 
 log = logging.getLogger(__name__)
 
-SilencedChannel = namedtuple(
-    "SilencedChannel", ("id", "ctx", "stop"))
+SilencedChannel = NamedTuple(
+    "SilencedChannel", [("ctx", Context), ("delay", int)]
+)
```

Comment:
```suggestion
class TaskData(t.NamedTuple): 
    """Data for a scheduled task.""" 
    delay: int 
    ctx: Context
)
```
This should be a class as it is clearer and can contain a docstring there